source/acquaman/AMStepScanActionController.cpp

```diff
@@ -118,13 +120,14 @@ void AMStepScanActionController::buildScanController()
 
 	QFileInfo fullPath(AMUserSettings::defaultRelativePathForScan(QDateTime::currentDateTime()));	// ex: 2010/09/Mon_03_12_24_48_0000   (Relative, and with no extension)
 
+	scan_->addAMDSFilePath(fullPath.filePath());
```

Comment:
No.
In AMScanController::setStarted(), we grab the file name and assigned it to the AMDSClientAppController instance. 
Then in AMContinuousScanActionController, we connected the scanAction signals with the AMDSClientAppController slots. Basically what we did by far is to open a file when scan started and close a file when scan stopped/cacelled/failed.

We didn't writing anything unless the acquman application requests data from AMDSClientAppController.


For beamline without AMDS, they will see an empty file created. That's all the side effect